packages/php/blueprint/src/ResourceStorages/OrgThemeResourceStorage.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\ResourceStorages;
+
+class OrgThemeResourceStorage extends OrgPluginResourceStorage {
+	protected function get_download_link( $slug ) {
+		if ( ! function_exists( 'themes_api' ) ) {
+			include_once ABSPATH . '/wp-admin/includes/themes.php';
+		}
+		$info = \themes_api(
+			'theme_information',
+			array(
+				'slug'   => $slug,
+				'fields' => array(
+					'sections' => false,
+				),
+			)
+		);
+
+		return $info->download_link;
```

Comment:
If the theme information is not found or an error occurs, `$info` might be false or an object without the download_link property.

Reply:
Good catch! Updated in [Check for download_link -- it can be empty](https://github.com/woocommerce/woocommerce/pull/49763/commits/3ff19aa5fd402bf6a400e37a103e7842fc2850b7)